package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "s47_bizarre_cravings",
+  "version": "1.0.0",
+  "description": "Hello",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
```

Comment:
Consider adding your name to the "author" field. This will help others know who the main contributors to the project are. [medium]